Source/Core/VideoCore.cpp

```diff
@@ -191,6 +191,24 @@ struct per_group VideoPerGroup [Group_VideoMax]=
         "of telecine). Each frame's classification is plotted with a half-life of 1.",
         ActiveFilter_Video_Idet,
     },
+    //Item_pkt_duration_time
+    {
+        Item_pkt_duration_time,     2,    0,    2,  4,  "pkt_duration_time", false,
+        "Plots an interpretation of the interlacement pattern of the visual image.\n"
```

Comment:
Change tooltip text to 

```
Plots the duration in seconds of each frame. If the file is of constant\n
frame rate than this should be a straight line.
```

Also split the two pieces of data into separate graphs. The tooltip for the `pkt_size` one could be:

```
Plots the size in bytes of each frame. If the file is of uncompressed\n
frame rate than this should be a straight line, but a lossless or\n
lossy file should show the variety of frame sizes.
```